File: iteration2/CSE3063F21P1_GRP16/src/main/java/Models/Person.java
package Models;

public abstract class Person{
    public String FirstName;
    public String LastName;
    private String PersonMail;
    private int ssn;

    public String getFirstName() {
        return FirstName;
    }

    public void setFirstName(String firstName) {
        FirstName = firstName;
    }

    public String getLastName() {
        return LastName;
    }
    public void setLastName(String lastName) {
        LastName = lastName;
    }

    public String getPersonMail() {
        return PersonMail;
    }

    public void setPersonMail(String personMail) {
        PersonMail = personMail;
    }

    public int getSsn() {
        return ssn;
    }

    public void setSsn(int ssn) {
        this.ssn = ssn;
    }

}

File: iteration3/CSE3063F21P1_GRP16/models/Person.py
from abc import ABC, abstractmethod

class Person(ABC):
    def __init__(self, first_name, last_name, person_mail, ssn):
        self.first_name = first_name
        self.last_name = last_name
        self._person_mail = person_mail
        self._ssn = ssn

    @abstractmethod
    def some_abstract_method(self):
        pass